tests/test_extension.py

```diff
@@ -33,3 +33,22 @@ def test_scan_cpu_equiv(samples: int, cmplx: bool):
     ext_y = torch.ops.torchlpc.scan_cpu(x, A, zi)
 
     assert torch.allclose(numba_y, ext_y)
+
+
+@pytest.mark.parametrize(
+    "samples",
+    [1024],
+)
+@pytest.mark.parametrize(
+    "cmplx",
+    [True, False],
+)
+def test_lpc_cpu_equiv(samples: int, cmplx: bool):
+    batch_size = 4
```

Comment:
You don't need to change this now, but in general I think it's worth parameterizing `batch_size` as well and testing batch_size = 1 and batch_size > 1 since this exposes `.squeeze()` and related errors.

Reply:
Fair. Will add it in the future.